textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToTrigram.java

```diff
@@ -0,0 +1,109 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import com.google.re2j.PublicParser;
+import com.google.re2j.PublicRE2;
+import com.google.re2j.PublicRegexp;
+import com.google.re2j.PublicSimplify;
+
+/**
+ * This class 
+ * {@link https://swtch.com/~rsc/regexp/regexp4.html}
+ * 
+ * @Author Zuozhi Wang
+ * @Author Shuying Lai
+ * 
+ */
+public class RegexToTrigram {	
```

Comment:
I think it's kind of different. This function is specifically used for RegexMatch. This algorithm will have several functions. They will be too long to be put in the Utils class.